sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubOptions.cs

```diff
@@ -132,6 +132,27 @@ public int MaxEventBatchSize
             }
         }
 
+        private int _targetUnprocessedEventThreshold;
+
+        /// <summary>
+        /// Get or sets the target unprocessed events per worker for eventhub functions. This is used in target based scaling to override the default behavior of using maxEventBatchSize as the target value.
+        /// If TargetUnprocessedEventThreshold is set, the total unprocessed event count will be divided by this value to determine the number of instances,
```

Comment:
```suggestion
        /// If TargetUnprocessedEventThreshold is set, the total unprocessed event count will be divided by this value to determine the number of worker instances,
```

nit:  You refer to "workers" above.  i think we need to be consistent here.

Reply:
Updated